CMakeLists.txt

```diff
@@ -12,20 +12,12 @@ set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${SC_EXTENSIONS_DIRECTORY})
 link_directories(${SC_BIN_PATH} ${SC_EXTENSIONS_DIRECTORY})
 
 set(SC_MACHINE_PATH "${CMAKE_CURRENT_SOURCE_DIR}/ostis-web-platform/sc-machine")
-set(SC_CODEGEN_TOOL "${SC_BIN_PATH}/sc-code-generator")
-set(CMAKE_MODULE_PATH "${SC_MACHINE_PATH}/cmake")
-include("${SC_MACHINE_PATH}/cmake/codegen.cmake")
+include(${SC_MACHINE_PATH}/CMakeLists.txt)
```

Comment:
Don't include CMakeLists.txt of sc-machine. You must add `include("${SC_MACHINE_PATH}/cmake/tests.cmake")`. Revert all changes